module-cheatsheets/intro-to-R-tidyverse-cheatsheet.md

```diff
@@ -109,14 +109,19 @@ Additional vignettes are available from the "Articles" dropdown menu on this web
 | `ggplot2`               | [`last_plot()`](https://ggplot2.tidyverse.org/reference/last_plot.html)           | Last plot                                       | Returns the last plot produced.                                           |
 
 
-#### `readr`, `tibble` and `tidyr`
-Read the [`readr` package documentation](https://readr.tidyverse.org/) and a [vignette on its usage](https://readr.tidyverse.org/articles/readr.html). <br>
-Read the [`tibble` package documentation](https://tibble.tidyverse.org/) and a [vignette on its usage](https://tibble.tidyverse.org/articles/tibble.html). <br>
+<div style="page-break-after: always;"></div>
+
+#### `readr`, `fs`, `tibble` `tidyr`
+Read the [`readr` package documentation](https://readr.tidyverse.org/) and a [vignette on its usage](https://readr.tidyverse.org/articles/readr.html).
+Read the [`fs` package documentation](https://fs.r-lib.org/).
+Read the [`tibble` package documentation](https://tibble.tidyverse.org/) and a [vignette on its usage](https://tibble.tidyverse.org/articles/tibble.html).
 Read the [`tidyr` package documentation](https://tidyr.tidyverse.org/) and a [vignette on its usage](https://tidyr.tidyverse.org/articles/tidy-data.html).
+https://fs.r-lib.org/reference/index.html
```

Comment:
I'm assuming this snuck in accidentally. 
```suggestion
```

(I don't think there is any need to include the `fs` comparison article: https://fs.r-lib.org/articles/function-comparisons.html

Reply:
yep, accident..